dvc/config.py

```diff
@@ -78,6 +78,7 @@ class Config(dict):
 
     # In the order they shadow each other
     LEVELS = ("system", "global", "repo", "local")
+    REPO_ONLY_LEVELS = ("repo", "local")
```

Comment:
I think it will make managing this in the future easier if we do something like
```suggestion
    SYSTEM_LEVELS = ("system", "global")
    REPO_LEVELS = ("repo", "local")
    LEVELS = SYSTEM_LEVELS + REPO_LEVELS
```

Reply:
Thanks @pmrowla. Even I thought about this but then later felt it maybe a too much thing for the simple fix. If you also feel the changes are good, I'll add those as well.